aqua/gimp2-launcher/files/patch-Info.plist.diff

```diff
@@ -13,7 +13,7 @@
      <key>NSHumanReadableCopyright</key>
      <string>© 2001-2022 The GIMP Team</string>
      <key>LSMinimumSystemVersion</key>
--	  <string>10.12</string>
+-	  <string>10.13</string>
```

Comment:
Just curious, why is this needed?

Reply:
Because the os version requirement of the app bundle changed. But since we build our own binary maybe we should remove this key altogether.